pyprland/plugins/scratchpads/__init__.py

```diff
@@ -531,8 +531,9 @@ async def run_show(self, uid) -> None:
     async def _show_transition(self, item, monitor, was_alive):
         "perfoms the transition to visible state"
         animation_type = item.conf.get("animation", "").lower()
+        close_special_workspace = item.conf.get("close_special_workspace")
```

Comment:
rather add the default value here (2nd parameter of get())

Reply:
The default value is set in `set_config`，so I think it should raise error when no such key.